src/Commands/SystemCommands/GenericmessageCommand.php

```diff
@@ -23,13 +24,36 @@ class GenericmessageCommand extends SystemCommand
     protected $name = 'Genericmessage';
     protected $description = 'Handle generic message';
     protected $version = '1.0.1';
+    protected $need_mysql = true;
     /**#@-*/
 
     /**
-     * {@inheritdoc}
+     * Execution if MySQL is required but not available
+     *
+     * @return boolean
      */
-    /*public function execute()
+    public function executeNoDB()
     {
+        //Do nothing
+        return true;
+    }
 
-    }*/
+    /**
+     * Execute command
+     *
+     * @return boolean
+     */
+    public function execute()
+    {
+        //System command, fetch command to execute if track exist
+        $message = $this->getMessage();
+        $chat_id = $message->getChat()->getId();
+        $user_id = $message->getFrom()->getId();
+        //Fetch Track if exist
+        $command = (new Conversation($user_id, $chat_id))->getConversationCommand();
+        if (! is_null($command)) {
+            return $this->telegram->executeCommand($command, $this->update);
+        }
+        return true;
```

Comment:
Great! I've added a PR to your branch to make use of the new method in the AdminCommand class. As soon as this is merged, you can use it in your conversations PR :+1: